<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dgCameraSelection</class>
 <widget class="QDialog" name="dgCameraSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>356</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>518</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btCamera1">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ФРОНТАЛЬНАЯ 
 КАМЕРА</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btCamera2">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ЗАДНЯЯ КАМЕРА</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btCamera3">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>КАМЕРА НА РУКЕ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btCamera4">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ОБЗОРНАЯ 
 КАМЕРА</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btCamera5">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Камера 5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btCamera6">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Камера 6</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
